index.php

```diff
@@ -56,3 +59,21 @@
 
 $mediaWiki = new MediaWiki();
 $mediaWiki->run();
+
+// SEO-586 Log what pages link to index.php URLs on 5% of page views
+if ( rand( 0, 100 ) < 5 ) {
+	$scriptUrl = $_SERVER[ 'SCRIPT_URL' ] ?? 'empty';
+	$scriptUri = $_SERVER[ 'SCRIPT_URI' ] ?? 'empty';
+	$queryString = $_SERVER[ 'QUERY_STRING' ] ?? 'empty';
+	$referrer = $_SERVER[ 'HTTP_REFERER' ] ?? 'empty';
+
+	if ( strstr( $scriptUrl , 'index.php' ) !== false
+		&& strstr( $scriptUrl , '/wiki/' ) == false
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)